content/en/Platform Deep Dive/Scans/faq.md

```diff
@@ -0,0 +1,72 @@
+The DAST scanner interacts with web pages similarly to how a human would, by clicking buttons and submitting forms on each crawled page. Do you have any critical pages or paths where you don't want the crawler to interact with buttons or submit forms? Examples include Settings, User Profiles, Security, Admin section, etc. Make sure the crawler is aware of which URLs to avoid to prevent unintended interactions and potential disruptions. You can add them to the Avoided URLs in the [target settings].
+
+## Can you give access to our developers only to DAST and not to the pentests?
+We don't have this fine-grained access control yet.
+
+## My scan seems completed but the status says "Under Review." What does this mean?
+'Under Review' status is an intermediate status before 'Completed.' If some vulnerabilities need to be manually confirmed by our team, the scan is set to Under Review; after this manual review, the scan changes to Completed.
+
+## The scan is completed but I see only one finding. Does this mean my app is secure? Or did the scanner fail?
+If the scan is completed, then the scanner didn't fail. However, you can check a few things:
+- Download the crawl report and check if the scanner visited all the pages you expected.
+- Make sure the scanner was able to log in to the app.
+- Ensure the test user has enough permissions to access all the pages and perform different actions.
+
+## Are you able to tailor vulnerability severities per target?
+We can't do that at the moment, but we have it as a future roadmap candidate. Due to some technical dependencies, we likely wouldn't be able to start on this soon.
+
+## Do you support scans against targets with basic HTTP auth enabled?
+We don't have this feature open to customers yet. However, you can contact us if you can't work around this limitation to assist you.
+
+## Does the DAST scanner support 2FA login?
+We don't have this feature open to customers yet. However, you can contact us if you can't work around this limitation to assist you.
+
+## Does the DAST scanner support CAPTCHA?
+We don't support this feature yet. Here are some suggestions:
+- Disable CAPTCHA for the scanner's IP address.
+- Set an exception for the test user in your system to bypass CAPTCHA checks.
+- Provide [custom headers] in the scanner settings to bypass CAPTCHA checks.
+
+## Does the DAST API scanner work with GraphQL APIs?
+Currently, we do not support standalone GraphQL API scanning. However, if there is a target with an interface that uses a GraphQL service as its backend API (e.g., SPA), then we can scan it. Standalone GraphQL API scanning is on our roadmap for future development.
+
+## We have two regional TLDs for our targets (for example, .de and .co.uk). Is this one target or multiple targets?
+If the domains are different, then they are considered different targets. You can add them as separate targets in the platform. Unlike /de and /co.uk, which are considered different paths of the same target.
```

Comment:
In 'Unlike /de and /co.uk' - shouldn't it be **.** instead of **/**?

Reply:
Here is it meant to be a path in the website, not a TLD.
How can I make it clearer? Maybe use `google.de` and `google.com/de`?

Reply:
Oh I see what's meant here then. Maybe in the answer part, we can include more complete examples. Like 'If the domains are different ('example.de' versus 'example.co.uk')...unlike 'example.com/de' and 'example.com/co.uk', which are considered different paths...'